opm/autodiff/StandardWell.hpp

```diff
@@ -220,6 +228,9 @@ namespace Opm
         // the saturations in the well bore under surface conditions at the beginning of the time step
         std::vector<double> F0_;
 
+        const RateConverterType& rateConverter_;
+        int pvtRegionIdx_;
```

Comment:
A question rather than comment.  `pvtRegionIdx_` here is for a well, while when calculating the mobility for a specific connection, the connection can have a different pvt region (specified in COMPDAT)?

Reply:
A connection can have different satnum region (used for the mobility and specified in COMPDAT) but hopefully not different pvt regions. At least from a physical sense it does not make much sense to have different pvt regions in cells with fluid communication.

Reply:
Thanks for the explanation, I messed that with the other thing, ``fs.pvtRegionIndex()`. which is related to the perforated cells? Do you think they can be different, or it is something difficult to say and better to avoid?

Reply:
Thanks for the explanation. Maybe it is better to keep `fs.pvtRegionIndex()` independent from this pvtRegionIdx_ for the moment unless tests show otherwise.